src/main/app/src/assets/i18n/nl.json

```diff
@@ -207,6 +207,7 @@
   "ontkoppeldDoor": "Ontkoppeld door",
   "ontkoppeldDoor.-kies-": "- Kies een medewerker -",
   "zoekopdracht": "Zoekopdracht",
+  "zaakOpschorten": "De zaak opschorten",
```

Comment:
Moet dit niet zijn zaak.opschorten ?

Reply:
Nee, deze laaten we gelijk aan de key van de taak variabele